README.md

```diff
@@ -1,9 +1,101 @@
 <a href="https://www.lacework.com/"><img src="https://www.lacework.com/wp-content/uploads/2019/07/Lacework_Logo_color_2019.svg" width="250px" height="100px" title="Lacework" alt="Lacework"></a>
 
-# Lacework Agent Release
-The source for officially supported Lacework agent releases
+# Lacework Agent Releases
+This Lacework agent GitHub site provides the officially supported Lacework agent releases. This site provides both the Lacework agent install and source files available for downloading.
```

Comment:
there are no sources available for download

Reply:
Ok. I took that out. I think it's very confusing for the customer to see **Source code (zip)**. That implies source code. 
![image](https://user-images.githubusercontent.com/48295842/88823910-7d805880-d17a-11ea-8708-8a82f909ee15.png)